docs/blockchain_schema/version1.ts

```diff
@@ -0,0 +1,61 @@
+// Version 1 schema
```

Comment:
Maybe have some prepending 0's before the 1 in version1 just so it can be changed and still end up sorting correctly :)

`version01.ts` :3 probably a long while before we hit 100 versions of the Blockchain lolz

---

Separately, I also think we should probably make it a subdirectory with some files inside, like for now

`version01/index.ts`

Because right now there isn't much you can put on the blockchain, but that will certainly grow.

Reply:
Also, can we just export everything in here so that they can be included and used elsewhere?